packages/replay/src/util/hasSessionStorage.ts

```diff
@@ -0,0 +1,6 @@
+import { WINDOW } from '../constants';
+
+/** If sessionStorage is available. */
+export function hasSessionStorage(): boolean {
+  return 'sessionStorage' in WINDOW && !!WINDOW.sessionStorage;
```

Comment:
l: do we even need the `in` check?

Reply:
TBH I was wondering the same thing, I vaguely remember that sometimes (??) `WINDOW.xxx` check may fail, but not 100% sure anymore under which circumstances that would happen 🤔 we use the `in` syntax for all our window checks, as far as I can tell...!

Reply:
Ahh I see. Better safe than sorry, was just wondering :)